addons/account/migrations/13.0.1.1/end-migration.py

```diff
@@ -21,7 +21,13 @@ def add_default_account_data(env):
              "default_cash_difference_income_account_id"),
         ]:
             chart = company.chart_template_id
-            account_template = chart[chart_field]
+            check_chart = chart
+            account_template = False
+            while check_chart:
+                account_template = check_chart[chart_field]
+                if account_template:
+                    break
+                check_chart = check_chart.parent_id
```

Comment:
Let's improve a bit the code/readability:
```suggestion
            account_template = False
            while not account_template and chart:
                account_template = chart[chart_field]
                chart = check_chart.parent_id
```

Reply:
I think your proposal can produce an endless while, right?

Reply:
No because the `and chart` part.

Reply:
I think we should stick with the proposal of @etobella , its strictly correct and super safe despite the readability.

Reply:
If you point to any problem in my proposal, then I'll accept the current one, but I see no drawback.

Reply:
ok, your proposal convinced me 👍

Reply:
Changes applied. Cannot reuse chart, as it is used later, but your idea was kept :wink: